adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationParameters.cs

```diff
@@ -147,14 +148,14 @@ private static async Task<AuthenticationParameters> CreateFromResourceUrlCommonA
             try
             {
                 IHttpClient request = new HttpClientWrapper(resourceUrl.AbsoluteUri, null);
-                using (await request.GetResponseAsync().ConfigureAwait(false))
```

Comment:
Minor refactoring - IHttpWebResponse does not need to implement IDisposable because it has nothing to dispose of.

Reply:
Does it contain any objects which does? E.g. the HttpClient?

---
In reply to: [217285444](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1255#discussion_r217285444) [](ancestors = 217285444)